rcl_logging_interface/test/test_get_logging_directory.cpp

```diff
@@ -0,0 +1,182 @@
+// Copyright 2020 Open Source Robotics Foundation, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <rcpputils/filesystem_helper.hpp>
+#include <rcpputils/get_env.hpp>
+#include <rcutils/allocator.h>
+#include <rcutils/env.h>
+#include <rcutils/error_handling.h>
+
+#include <string>
+
+#include "gtest/gtest.h"
+#include "rcl_logging_interface/rcl_logging_interface.h"
+
+// This is a helper class that resets an environment
+// variable when leaving scope
+class RestoreEnvVar
+{
+public:
+  explicit RestoreEnvVar(const std::string & name)
+  : name_(name),
+    value_(rcpputils::get_env_var(name.c_str()))
+  {
+  }
+
+  ~RestoreEnvVar()
+  {
+    if (!rcutils_set_env(name_.c_str(), value_.c_str())) {
+      std::cerr << "Failed to restore value of environment variable: " << name_ << std::endl;
```

Comment:
Nit: `#include <iostream>` at the top for this.

Reply:
fixed in 49e71a001b41deeaf29a7baf6e2444c181855da7